src/pyconverter/xml2py/cli.py

```diff
@@ -50,6 +56,14 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
         The default value is the ``_package`` directory accessible in the
         `PyConverter-XML2Py GitHub repository <https://github.com/ansys/pyconverter-xml2py/tree/main/_package>`_.
 
+    run_black: bool, optional
+        Whether to run black CLI on the autogenerated package source code.
```

Comment:
```suggestion
        Whether to run Black CLI on the autogenerated package source code.
```
I checked their doc, and they use an initial capital letter in references to the product name.